src/containers/statistics/Statistics.js

```diff
@@ -1,11 +1,77 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Statisitcs = () => {
-  return (
-    <div>
-      <h3>Aggregated statistics</h3>
-    </div>
-  );
+/* eslint-disable no-unused-vars */
+const allEmployees = (data) => {
+  const allEmployees = [];
+  data.forEach((comp) => {
```

Comment:
Use map instead of empty array and foreach : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map